tests/testing/test_results.py

```diff
@@ -45,15 +43,13 @@ def test_simple_defaults(self):
                     "Dimension mismatch: ('dim_0', 'dim_1') != ('dim_0',)",
                     "Coordinates mismatch for dimension 'dim_0': [0 1] != [0 1 2]",
                     "Shape mismatch: (2, 3) != (3,)",
-                    dirty_equals.IsStr(regex="Left and right DataArray objects are not close.*", regex_flags=re.DOTALL),
                 ],
             ),
             (
                 xarray.DataArray([[1], [2], [3]]),
                 [
                     "Dimension mismatch: ('dim_0', 'dim_1') != ('dim_0',)",
                     "Shape mismatch: (3, 1) != (3,)",
-                    dirty_equals.IsStr(regex="Left and right DataArray objects are not close.*", regex_flags=re.DOTALL),
```

Comment:
I don't completely understand at the moment why these issues are not triggered anymore with the new implementation